sentences/de/light_HassLightSet.yaml

```diff
@@ -24,3 +24,12 @@ intents:
           - "<setzen> die Farbe [<lichter>|<alle_lichter>] [<area>] auf {color}"
           - "die Farbe [<lichter>|<alle_lichter>] [<area>] auf {color} <setzen>"
         response: color_area
+
+        # Max/Min brightness
+      - sentences:
+          - "<setzen> [[[die] Helligkeit [von|von der|von den|der] ]| [die|das]] <name> auf [die|das] {brightness_level:brightness}  [ein]"
```

Comment:
we can use the `<von_dem>` expansion rule here, further the `<name>` rule does already include articels